parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -242,7 +242,7 @@ pub mod pallet {
 		pub(crate) fn commit_messages() {
 			let count = MessageLeaves::<T>::decode_len().unwrap_or_default() as u64;
 			if count == 0 {
-				return
+				return;
```

Comment:
Ditto, I suspect your formatter may be misconfigured.

Reply:
Fixed in https://github.com/Snowfork/snowbridge/pull/972/commits/cc9b1fc2aa6f50764287be80db7eb4e8666d4fe7.